submissions/NatalyaBortsova/popup/index.html

```diff
@@ -0,0 +1,175 @@
+                      <img src="images/icons/contacts.png" alt="Go to Contacts page" width="50" height="50" />
+                      <span class="popup__name">Contacts</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/documents.png" alt="Go to Documents page" width="50" height="50" />
+                      <span class="popup__name">Documents</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/files.png" alt="Go to Files page" width="50" height="50" />
+                      <span class="popup__name">Files</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/mail.png" alt="Go to Mail list" width="50" height="50" />
+                      <span class="popup__name">Mail</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/maps.png" alt="Go to Maps page" width="50" height="50" />
+                      <span class="popup__name">Maps</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/mars.png" alt="Go to Mars page" width="50" height="50" />
+                      <span class="popup__name">Mars</span>
+                    </a>
+                  </li>
+                  <li class="popup__item">
+                    <a href="#" class="popup__link animation animation_icon">
+                      <img src="images/icons/news.png" alt="Go to News page" width="50" height="50" />
+                      <span class="popup__name">News</span>
+                    </a>
+                  </li>
+                </ul>
+                <div class="popup__block">
+                  <input class="menu__item-checkbox menu__item-checkbox_more" id="more" type="checkbox">
+                  <label class="menu__item-label menu__item-label_more" for="more">More</label>
+                  <ul class="popup__list popup__list_more">
+                    <li class="popup__item">
+                      <a href="#" class="popup__link animation animation_icon">
+                        <img src="images/icons/photos.png" alt="Go to Photos page" width="50" height="50" />
+                        <span class="popup__name">Photos</span>
+                      </a>
```

Comment:
The main point of the article is no need to populate `alt` attribute for decorative images. The icon is a decorative image. You must delete `alt` attribute values)